packages/compat/src/__tests__/keypair.ts

```diff
@@ -0,0 +1,4 @@
+// TODO: Not totally sure how to test this yet
+describe('keypair', () => {
+    it.todo('should convert from a Legacy Web3 JS Keypair to a CryptoKeyPair');
+});
```

Comment:
I'd probably just test the same as what the `generateKey` polyfill tests - basically that the properties on the key are set correctly
https://github.com/solana-labs/solana-web3.js/blob/5e1b2e5d451f4016b1d846d84b3fa3309e43e494/packages/webcrypto-ed25519-polyfill/src/__tests__/secrets-test.ts#L55-L83